Kernel/CMakeLists.txt

```diff
@@ -342,17 +335,25 @@ set(KERNEL_SOURCES
     TTY/SlavePTY.cpp
     TTY/TTY.cpp
     TTY/VirtualConsole.cpp
+    Tasks/Coredump.cpp
```

Comment:
this file name might be too generic if itʼs related strictly to the tasks (or it just does not belong there?). maybe `Debug` is a place to consider? 🤔

Reply:
Coredump is a thing that is related to tasks (processes), so we should keep it there.

Reply:
but also with debugging them? as you did not moved `Debug.cpp`, it makes sense. 👍